examples/iam-github-oidc/main.tf

```diff
@@ -21,6 +21,12 @@ locals {
 module "iam_github_oidc_provider" {
   source = "../../modules/iam-github-oidc-provider"
 
+  # https://github.blog/changelog/2023-06-27-github-actions-update-on-oidc-integration-with-aws/
+  additional_thumbprint_list = [
```

Comment:
Instead of putting these values in the examples, we would rather have them as defaults, so users should just upgrade the version of the module and use it right away.

/cc @bryantbiggs WDYT?

Reply:
ya we can try that. I've checked internally as well and there isn't a clear path to resolution so lets try and see what happens